Signal/src/call/CallService.swift

```diff
@@ -304,6 +324,10 @@ protocol CallServiceObserver: class {
                 throw CallError.obsoleteCall(description:"obsolete call in \(#function)")
             }
 
+            // For outgoing calls, wait until call offer is sent before we send any ICE updates, to ensure message ordering for
+            // clients that don't support receiving ICE updates before receiving the call offer.
+            self.readyToSendIceUpdates(call: call)
+
```

Comment:
This is partially redundant with the re-introduced "wait for CallAnswer before sending Ice updates" but we'll want to keep this part of the solution in place even when we get rid of the "wait for CallAnswer" behavior to ensure message ordering.